index.rst

```diff
@@ -6,24 +6,119 @@
+
+
+Table1: Navigating between actuator IDs and sequential indices.
+
+Bump Test States
+==================================
+
+As the bump test cycles through the testing, the EFD location:
+
+| lsst.sal.MTM1M3.logevent_forceActuatorBumpTestStatus
+|
+
+returns where in the bump test cycle the test is.  Figure 5 shows the progression of these states.  At the conclusion of the test for each actuator, the state will either be 6 (PASSED) or 7 (FAILED).
+
+.. image:: ./_static/Bump_Test_States.png
+
+Figure 5. This shows the bump test states as logged in  lsst.sal.MTM1M3.logevent_forceActuatorBumpTestStatus.
+
+XYZ forces vs Cylinder forces
+==============================
+
+The force data in the EFD includes both the forces applied to the primary and secondary cylinders, as well as the forces in the X, Y, and Z directions.  These forces are not the same, as will be explained in this section.  Referring to Figure 6, we see that the primary cyclinder applies force in the axial direction, which is along the optical axis of the telescope, and is referred to as the Z direction.  So the force applied by the primary cylinder is equal to the Z-force.  The secondary cylinder however, is at a 45 degree angle to the Z-axis.  So if we want to apply a force in the lateral direction (X or Y), we need to apply a force with the secondary cylinder, and then a negative force with the primary cylinder so that the resultant is in the lateral direction.
+
+.. image:: ./_static/Force_Schematic.png
+
+Figure 6. This schematic of the actuators shows the relation between the cylinder forces and the resulting axial and lateral forces.
+
+The primary and secondary forces are what is actually measured, and the X, Y, and Z forces are calculated from these forces and the known geometry.  So Figures 3 and 4 show the Y and Z forces.  Figures 7 and 8 show the target and actual forces in the primary and secondary cylinders.  Note the negative primary force being applied during the secondary bump test.
+
+.. image:: ./_static/Bump_Test_Cylinder_Target.png
+
+Figure 7.  This shows the target forces to be applied to the primary and secondary actuators during the bump test of a single actuator.
+
+.. image:: ./_static/Bump_Test_Cylinder_Results.png
+
+Figure 8. This shows the actual measured forces applied by the primary and secondary actuators during the bump test of a single actuator
+
+Bump Test Following Errors
+==============================
+
+We can also plot the bump test following errors, which are the difference between the target force and the actual measured force.  The actuator is not able to follow the step function in the applied force, so there is large spike in the following error after a change in the target force.  For this reason, the plot in Figure 9 uses the "symlog" scale, which is linear between -10 Nt and +10 Nt, and log above +/-10 Nt.  This allows one to see the errors in the relatively flat region of the test, which is most important.
```

Comment:
Levels and times for absolute value of the following error are specified in M1M3 SS CSC configuration file (ts_config_mttcs/MTM1M3/v1/_init.yaml):

```
  BumpTest:
    TestedTolerances:
      Warning: 2.5
      Error: 5
    NonTestedTolerances:
      Warning: 20
      Error: 100
    SettleTime: 3.0
    Measurements: 10
```

Where non-tested mean all other FAs (the algorithm check if all other except for tested cylinder doesn't show significant force), tested are error and warning levels for cylinder being tested. The algorithm waits up to SettleTime seconds to see "Measurements" number of measured absolute values dropping below errorr level for FA to pass the test. If measured values venture above "warning" level, but stay within "error" level, a warning is send into M1M3 log.